index.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-console */
+import { Client, ClientSocket, NodeMessage } from 'veza';
+import { TextChannel } from 'discord.js';
+import Cluster from './packages/typicalbot/src';
```

Comment:
This file should be moved to /packages/typicalbot and should be used to start the client.

Reply:
This can't be moved till we rework the clusters.

Reply:
I think it could be moved, unless the cluster somehow depends on this file being located in the root directory, though nothing in this file seems to suggest that it would. Lines 4 and 5 would just need to be updated to reflect the path change.

Reply:
I did try and move the file but it continues throwing the oddest error that I was unable to debug and see why it was occurring. I added so many console.logs line by line to try and find the cause but everything seemed to be working properly but trying to move the file was the only thing that would cause the issue. Undoing it removed the errors and allowed the bot to start.

Reply:
As I said, this file cannot be moved, yet. It will be moved/removed in a future version.